core/src/main/resources/db/migration.sql

```diff
@@ -191,3 +191,83 @@ CREATE TABLE `structural_variant` (
   FOREIGN KEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE
 );
 UPDATE info SET DB_SCHEMA_VERSION="1.3.1";
+
+
+##version: 1.3.2
+-- ========================== start of genetic_entity related migration =============================================
+-- add genetic_entity table 
+CREATE TABLE `genetic_entity` (
+  `INTERNAL_ID` int(11) NOT NULL AUTO_INCREMENT,
+  `ENTITY_TYPE` varchar(45) NOT NULL,
+  PRIMARY KEY (`INTERNAL_ID`)
+);
+-- update gene table to use genetic_element:
+ALTER TABLE `gene` 
+ADD COLUMN `GENETIC_ENTITY_ID` INT NULL FIRST;
+
+-- add temporary column to support migration:
+ALTER TABLE `genetic_entity` 
+ADD COLUMN `TMP_GENE_ID` INT NOT NULL AFTER `ENTITY_TYPE`,
+ADD UNIQUE INDEX `TMP_GENE_ID_UNIQUE` (`TMP_GENE_ID` ASC);
+
+-- populate genetic_entity
+insert into genetic_entity (entity_type, tmp_gene_id)
+(Select 'GENE', ENTREZ_GENE_ID from gene);
+
+-- update gene table to have GENETIC_ENTITY_ID point to the correct one:
+SET SQL_SAFE_UPDATES = 0;
```

Comment:
yeah...although I think this might only be needed when running in mysql directly. I still need to test running this via the python script. Will do.